app/models/project_step.rb

```diff
@@ -113,7 +130,7 @@ def status
   end
 
   def display_date
-    I18n.l (self.completed_date || self.scheduled_date), format: :long
+    I18n.l (self.completed_date || self.scheduled_start_date), format: :long
```

Comment:
Don't think you need the parens here, and they confused me at first. Also don't need the `self`. Avoid `self` unless it's needed (like in assignment).